Achilles/Achilles.pde

```diff
@@ -1,15 +1,18 @@
-Laser l;
+World world;
 
 void setup() {
-  fullScreen();
+  size(800, 600);
```

Comment:
Fullscreen was hard to work with on my laptop 😛 but we can add it back in whenever it's a simple change.